FreeAPS/Sources/Modules/Settings/View/TidepoolStartView.swift

```diff
@@ -0,0 +1,43 @@
+
+import SwiftUI
+
+struct TidepoolStartView: View {
+    @ObservedObject var state: Settings.StateModel
+
+    var body: some View {
+        Form {
+            Section {
+                Text("Connect to Tidepool")
+                    .onTapGesture {
+                        state.setupTidepool = true
+                    }
+
+            } footer: {
+                Text(
+                    "When connected, uploading of carbs, bolus, basal and glucose from Trio to your Tidepool account is enabled. \n\nUse your Tidepool credentials to login. If you dont already have a Tidepool account, you can sign up for one on the login page."
```

Comment:
Adding two Text elements here would probably make translations easier and should provide basic styling (space between elements) out of the box.

Reply:
Yes. Will fix that 😊

Reply:
Now fixed